README.md

```diff
@@ -62,20 +63,21 @@ k3a-lag-exporter {
-| poll-interval             | How often to poll the Kafka cluster                                                                                                   |
-| reporters.prometheus.port | Port of built-in Prometheus web server                                                                                                |
-| clusters                  | List of clusters to monitor. Currently, only a single cluster is supported.                                                           |
-| name                      | The cluster name. Will be used as a label in the exported metrics.                                                                    |
-| topic-allow-list          | Optional list of topics to include. See below.                                                                                        |
-| topic-deny-list           | Optional list of topics to exclude. See below.                                                                                        |
-| group-allow-list          | Optional list of consumer groups to include. See below.                                                                               |
-| group-deny-list           | Optional list of consumer groups to exclude. See below.                                                                               |
-| bootstrap-servers         | Kafka server(s) to connect to.                                                                                                        |
-| consumer-properties       | Properties allowing connection to the Kafka cluster as a consumer. Must have DESCRIBE permissions for the cluster, groups and topics. |
-| admin-properties          | Properties allowing connection to the Kafka cluster as an admin. Must have DESCRIBE permissions for the cluster, groups and topics.   |
+| Element                               | Description                                                                                                                           |
+|---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------|
+| k3a-lag-exporter                      | The main configuration object.                                                                                                        |
+| poll-interval                         | How often to poll the Kafka cluster. Default: 30 seconds                                                                              |
+| reporters.prometheus.port             | Port of built-in Prometheus web server. Default: 8000                                                                                 |
+| reporters.prometheus.metric-namespace | The prefix for Prometheus metrics. Default: k3a                                                                                       |
```

Comment:
Is is possible somehow to refer to where the defaults are mastered. I prefer to avoid repeating this, as it's error-prone.

Reply:
Sure. Better now?

Reply:
Yes! But what has changed inside the table? Table changes are hard to review.

Reply:
Easier using the [rich diff](https://github.com/statnett/k3a-lag-exporter/pull/31/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5)?